Samples/src/PhaseEstimation/BayesianPhaseEstimation.qs

```diff
@@ -296,129 +282,123 @@ namespace Microsoft.Quantum.Samples.PhaseEstimation {
     /// - For the theoretical and algorithmic background see
     ///   [ Page 1 of arXiv:1508.00869 ](https://arxiv.org/pdf/1508.00869.pdf#page=1)
     operation BayesianPhaseEstimation (nGridPoints : Int, nMeasurements : Int, oracle : ((Double, Qubit[]) => Unit is Ctl), eigenstate : Qubit[]) : Double {
-        
+
         // Initialize a grid for the prior and posterior discretization.
         // We'll choose the grid to be uniform.
         let dPhase = 1.0 / IntAsDouble(nGridPoints - 1);
         let maxTime = 100.0;
         mutable phases = new Double[nGridPoints];
         mutable prior = new Double[nGridPoints];
-        
+
         for (idxGridPoint in 0 .. nGridPoints - 1) {
             set phases w/= idxGridPoint <- dPhase * IntAsDouble(idxGridPoint);
             set prior w/= idxGridPoint <- 1.0;
         }
-        
+
         // We can now check that we get a prior estimate of about 0.5
         // by integrating φ over the prior defined above.
-        let priorEst = Integrate(phases, MultiplyPointwise(phases, prior));
+        let priorEst = Integrated(phases, PointwiseProduct(phases, prior));
         Message($"̂φ from prior: {priorEst}. Should be approximately 0.5.");
-        
+
         // Having assured ourselves that the prior is a reasonable
         // approximation to the true prior, we can now proceed to take
         // actual measurements using phase estimation iterations.
         for (idxMeasurement in 0 .. nMeasurements - 1) {
-            
+
             // Pick an evolution time and perturbation angle at random.
             // To do so, we use the RandomReal operation from the canon,
             // asking for 16 bits of randomness.
             let time = PowD(9.0 / 8.0, IntAsDouble(idxMeasurement));
-            
+
             // Similarly, we pick a perturbation angle to invert by.
             let inversionAngle = RandomReal(16) * 0.02;
-            
+
             // Now we actually perform the measurement.
             let sample = IterativePhaseEstimationStep(time, inversionAngle, oracle, eigenstate);
-            
+
             // Next, we calculate the likelihood
-            
+
             //     Pr(One | φ; t) = sin²([φ - θ] t / 2)
-            
+
             // for the new sample, where φ is the unknown phase, θ is the
             // inversion angle applied above, and where t is the evolution
             // time. The likelihood for observing Zero is similar, with
             // cos² of the argument instead of sin².
-            
+
             // We calculate the likelihood at each phase in our
             // approximation of the prior.
             mutable likelihood = new Double[nGridPoints];
-            
+
             if (sample == One) {
-                
                 for (idxGridPoint in 0 .. Length(likelihood) - 1) {
```

Comment:
Possible use of `IndexRange` library function.

Reply:
Fixed here as well, thanks!